arch-bootstrap.sh

```diff
@@ -108,8 +108,7 @@ configure_minimal_system() {
   local DEST=$1
   
   mkdir -p "$DEST/dev"
-  echo "root:x:0:0:root:/root:/bin/bash" > "$DEST/etc/passwd" 
-  echo 'root:$1$GT9AUpJe$oXANVIjIzcnmOpY07iaGi/:14657::::::' > "$DEST/etc/shadow"
+  sed -ie 's/^root.*/root:$1$GT9AUpJe$oXANVIjIzcnmOpY07iaGi\/:14657::::::/' $DEST/etc/shadow
```

Comment:
If you're sure you want to use `^root.*` instead of something a little more robust, then this LGTM.